common/models/src/auth/user.rs

```diff
@@ -180,6 +180,10 @@ impl UserOptions {
     pub fn hidden_password(&mut self) {
         self.hash_password.replace("*****".to_string());
     }
+
+    pub fn change_password(&mut self) {
```

Comment:
I think the name is ambiguous and could be changed to "set_must_change_password(&mut self, must_change_password: bool)"